scripts/src/main/resources/scripts/command/tomcat

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup version help"
+  fi
+  exit
+fi
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+TOMCAT_HOME="${DEVON_IDE_HOME}/software/tomcat"
+TOOL_VERSION_COMMAND="tomcat version"
+# shellcheck source=scripts/commandlet-cli
+source "$(dirname "${0}")"/../commandlet-cli
+
+# Call tomcat with specified arguments.
+function doRun() {
+  doSetup silent
+  doRunCommand "${TOMCAT_HOME}/bin/tomcat ${*}"
+}
+
+# $1: optional silent mode
+function doSetup() {
+  doInstall "tomcat" "${TOMCAT_VERSION}" "${1}"
+  return 0
+}
+
+# CLI
+case "${1}" in 
+"help" | "-h")
+  echo "Install Tomcat."
+  echo
+  echo "Arguments:"
+  echo " setup          install tomcat on your machine."
+  echo " «args»         call tomcat with the specified arguments. Call tomcat --help for details or use tomcat directly as preferred."
+;;
+"setup" | "s" | "")
+  doSetup "${2}"
+;;
+*)
+  doRun "${@}"
+;;
```

Comment:
there also seems to be a `startup` and a `shutdown` script in order to start and stop tomcat.
It can also be added later but it would be nice to have support for `devon tomcat start` and `devon tomcat stop`.